cluster/gce/gci/configure-helper.sh

```diff
@@ -2124,8 +2123,8 @@ function setup-etcd-encryption {
   echo "${ENCRYPTION_PROVIDER_CONFIG}" | base64 --decode > "${encryption_provider_config_path}"
   kube_api_server_params+=" --encryption-provider-config=${encryption_provider_config_path}"
 
-  default_encryption_provider_config_vol=$(echo "{ \"name\": \"encryptionconfig\", \"hostPath\": {\"path\": \"${encryption_provider_config_path}\", \"type\": \"File\"}}" | base64 | tr -d '\r\n')
```

Comment:
Does this leave newlines in the output if you yank out the `|tr -d '\r\n'`?

Reply:
Yes. 
~$ echo "foo" | base64 
Zm9vCg==
~$ 

The "tr -d" was there to make this code run properly on Macs - I don't think this is a requirement anymore.

Reply:
If your input is long enough, you will generate a base64 string that won't decode properly:

```
$ printf 'A%.0s' {1..100} | base64
QUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFB
QUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQQ==

$ foo=$(printf 'A%.0s' {1..100} | base64)

$ echo $foo | base64 -d
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAbase64: invalid input
```
`base64 -w` fixes this on Linux, but the Darwin version doesn't have the same flag.

Reply:
Thank you for catching this.
Added "-w0" flag.

Reply:
Is there a reason not to just leave the `|tr -d '\r\n'`? Did it break something?

Reply:
No, this does not break anything.
Removing only to simplify .
As I mentioned, this was added to support Macs. I doubt we still need this.